src/dparse/parser.d

```diff
@@ -3123,10 +3105,7 @@ class Parser
         if (isTemplate && currentIs(tok!"if"))
             mixin(parseNodeQ!(`node.constraint`, `Constraint`));
 
-        if (currentIs(tok!";"))
```

Comment:
`parseFunctionBody` returns if the first token is a semicolon. I removed here because `parseFunctionBody` may be called by hand at some time and for some reasons